build.gradle.kts

```diff
@@ -1,16 +1,16 @@
 description = "A Cross-Component Issue Management System for Component-based Architectures"
 
 plugins {
-    id("org.springframework.boot") version "2.7.0"
-	kotlin("jvm") version "1.6.21"
-	kotlin("plugin.spring") version "1.6.21"
+	kotlin("jvm")
 }
 
-repositories {
-    mavenCentral()
+allprojects {
+    repositories {
+        mavenCentral()
+        mavenLocal()
```

Comment:
do I even have to write anything?

Reply:
removed with bd06f4fc1d6e3582e70f6ba356be9840bb764465